docs/tutorials/tabular_prediction/tabular-feature-engineering.md

```diff
@@ -0,0 +1,184 @@
+
+:label:`sec_feature_engineering`
```

Comment:
For some reason on the side-bar, every section of this tutorial is listed. I'm unsure why this is happening, but we ideally only want it to be 1 item on the list.

The following might fix:

```
# Feature Engineering
:label:`sec_feature_engineering`
```

Reply:
Have updated as suggested.